src/utils/get-featured-card-fields.js

```diff
@@ -0,0 +1,30 @@
+import dlv from 'dlv';
+
+export const getFeaturedCardFields = article => {
+    if (!article) {
+        return {
+            image: null,
+            slug: null,
+            title: null,
+            description: null,
+            tags: null,
+        };
+    }
+    const query_fields = article.query_fields;
+    const image = query_fields['atf-image'];
+    const slug = query_fields.slug;
+    const title = dlv(query_fields, ['title', 0, 'value']);
+    const description = dlv(query_fields, [
```

Comment:
Can this use get-nested-value?

Reply:
Is there a specific rule for using `getNestedValue` vs `dlv`? If it is a node should we default to `getNestedValue`?

Reply:
Yea, that's what I'm thinking. There is a chance for there to be intermediate nodes we don't expect if there is a formatting error in the RST or a writer gets fancy.